### ISSUE TITLE
Getting "Could not retrieve order" error when trying to pay with Paypal and missing "Advanced Card Processing" option

### ISSUE THREAD

**Issue by haritslamis71** [woocommerce/woocommerce-paypal-payments#1437]
### Describe the Bug

All orders since last night failed with "Could not retrieve order" error (screenshot below) when submitting an order and we tried it ourselves few times just now and got the same issue.

Additionally, we also notice that the "Advanced Card Processing" option is no longer available on our checkout page.

### To Reproduce

1. Add item to cart
2. checkout and select Paypal as payment method -- as you can see on the screenshot, there is no longer have "Advanced Card Processing" option 
3. Click on Place order

### Screenshots
![image](https://github.com/woocommerce/woocommerce-paypal-payments/assets/136631680/89f7e984-a068-46b7-90e9-39b1776b7f74)

### Expected Behavior

1. No error when placing an order and should go to paypal
2. There should be "Advanced Card Processing" option

### Environment

 - WordPress Version: 6.2.2
 - WooCommerce Version: 7.8.0
 - Plugin Version: 2.1.0
 - Browser [e.g. Chrome, Safari] and Version: Edge, chrome
 - Any other plugins installed: 
 - 
![image](https://github.com/woocommerce/woocommerce-paypal-payments/assets/136631680/0d47933e-4551-451e-9726-8a31ce127a11)

![image](https://github.com/woocommerce/woocommerce-paypal-payments/assets/136631680/78dd9d95-2665-456c-a267-acf51e003c20)

### Additional Details

<details><summary>System status</summary>

  ```
  <!--If applicable, paste the system status here. Please ensure you redact or remove any identifying information. -->
  ```

</details>

**Comment by Sirvijver** [comment]
Hello @haritslamis71 

You have reached out to open GitHub forum with a support request. We utilize GitHub mainly for development activities, so I would like to ask you to [reach out to the support team](https://woocommerce.com/document/woocommerce-paypal-payments/#get-help) directly for more guidance.

The issue you have encountered is usually caused by the JavaScript that is used to replace the WooCommerce button with our PayPal button, is being blocked. Since we don't have the URL to your site we cannot really verify this; please add this when/if you open a support ticket via the provided link.
What you can do is to do conflict testing; deactivating all plugins (except WooCommerce and PayPal) and replacing the theme with the [default theme Storefront](https://wordpress.org/themes/storefront/) to see if the behavior persists.

Here's a guide that explains the steps in more detail: [How to test for conflicts](https://woocommerce.com/document/how-to-test-for-conflicts/). Please let us know how this works out for you.

You might want to check what has changed since yesterday, since I don't expect this to be caused by the update we just released (version 2.1.0). The conflict testing should clear this up.
You might also want to deactivate and activate the plugin itself to see if the Advanced Card Processing payment method appears again. Sometimes the update causes this issue, and reactivating the plugin verifies the account when being reactivated again.

If none of these helps and you require additional support, please open a support ticket via the instructions provided above.

Kind regards, 
Joost

**Comment by haritslamis71** [comment]
Thanks Joost
